app/scripts/react-directives/components/modals/formModal.jsx

```
@@ -12,18 +12,27 @@
 import { observer } from 'mobx-react-lite';
 
 const FormModal = observer(({ id, active, title, formStore, onOk, onCancel, okButtonLabel }) => {
+  const formId = id + 'Form';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)